boards/nrf51dk/src/io.rs

```diff
@@ -48,3 +47,59 @@ macro_rules! println {
         ($fmt:expr) => (print!(concat!($fmt, "\n")));
             ($fmt:expr, $($arg:tt)*) => (print!(concat!($fmt, "\n"), $($arg)*));
 }
+use core::fmt::{Write, write};
+use core::fmt::Arguments;
```

Comment:
It seems like I am able to run it, as is. How ever, changing it since it probably is a better way of writing code.